python/nuke/layer_alchemy/utilities.py

```diff
@@ -7,13 +7,21 @@
 
 import constants
 
+if nuke.GUI:
```

Comment:
You should probably refactor this code somewhere, or maybe use [Qt.py](https://github.com/mottosso/Qt.py)

Reply:
created a separate python file for it, and all imports are now only de-facto executed by the user in GUI mode (menu items and buttons). Will definitely pull in Qt.py if more Qt finds it's way in the project.